plugins/content_distribution/providers/youtube_api/lib/YoutubeApiDistributionEngine.php

```diff
@@ -315,6 +315,11 @@ protected function doSubmit(KalturaDistributionSubmitJobData $data, KalturaYoutu
 		
 		return $distributionProfile->assumeSuccess;
 	}
+
+	private function sanitizeFromHtmlTags($filed)
+	{
```

Comment:
Since this function does not access an object (not access to this) make it static